### DISCUSSION TITLE
Black Rhinos - Başvurular (Kaptan Berke)

### DISCUSSION THREAD

**Discussion by Berkek007** [sonsuzus/BilsemPython#67] [Genel]
Katılmak isteyenler yazabilir.

![Estd 1999](https://user-images.githubusercontent.com/114423221/209968107-fd907394-05bf-44b1-99d6-2c766fc19e04.png)

**Comment by OmerAkbayrak** [comment]
**Ömer Akbayrak** ÖYG 2'yim salı günleri 3-4. derslere giriyorum bu gruba katılmak istiyorum

**Reply by Berkek007** [reply]
Merhaba Ömer takıma katılabilirsin. Numaranı yazarsan Whatsapp'tan seni grubumuza ekleyebilirim.

**Reply by OmerAkbayrak** [reply]
Teşekkürler
05306792953

**Reply by OmerAkbayrak** [reply]
Ekleyebilirsin

**Comment by AhmetEmirErsoz** [comment]
*Ahmet Emir Ersöz* ÖYG 1 çarşamba 1-2 derslere geliyorum gruba katılmak istiyorum

**Reply by Berkek007** [reply]
Merhaba Ahmet sen de katılabilirsin numaran bende zaten var gruba ekledim.

**Comment by ghost** [comment]
Ali İlker Can ÖYG cuma 1-2 derslere geliyorum gruba katılmak istiyorum

Numara:
05411483431

**Comment by Atakanoyg1** [comment]
merhaba katılmak istiyorum adım Atakan ŞAHİN ÖYG 1 Çarşamba 3. ve 4. dersler

**Comment by Alperg123456** [comment]
Merhaba katılmak istiyorum adım Alper Gürel ÖYG 2'yim Perşembe günü geliyorum.

**Reply by Berkek007** [reply]
Gelebilirsin Alper iyi çalışmalar, gruptasın zaten

**Comment by onursonkaya** [comment]
Black Rhinosa katılmak istiyorum

**Reply by Berkek007** [reply]
Katılabilirsin Onur, takıma hoş geldin.

**Comment by Muhammedipek** [comment]
Berke Ben Muhammed İpek Öyg 2 öğrencisiyim perşembe son iki ders grubundayım takıma yararlı olacağımı düşünüyorum

**Reply by Berkek007** [reply]
Merhaba, Muhammed takıma katılabilirsin, hoş geldin.
Numaranı yazarsan WhatsApp grubuna ekleyeceğim.

**Reply by Muhammedipek** [reply]
05057170498
